.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         version:
           - '1.5'
-          - 'nightly'
+          # - 'nightly'
```

Comment:
Thanks, that was annoying

Reply:
Agreed! Thanks for doing this. Since 1.6.7 is the current long-term support release and 1.8.2 is the current stable release, maybe we should add Julia tests for versions 1.6, 1.7, and 1.8? We should probably also think about bumping the Julia version from 1.5 to 1.6 at some point, so adding the test would be a helpful first check.

Reply:
Sounds good. Let's tackle that in a separate PR. Do you want to take care of it @lanesmith?